### PR TITLE
added option to create accounts file for txgen

### PR DESCRIPTION
**Pull request by bogdan-rosianu**

### CODE REVIEW

File: cmd/filegen/main.go [hunk 1]
```diff
@@ -73,6 +74,11 @@ VERSION:
 		Usage: "Metachain consensus group size",
 		Value: 15,
 	}
+	numAdditionalAccountsInGenesis = cli.IntFlag{
```
Comment (AdoAdoAdo) [inline cmd/filegen/main.go]: not clear, why additional?
what will these additional accounts be used for?

File: cmd/filegen/main.go [hunk 2]
```diff
@@ -97,13 +107,23 @@ VERSION:
...
 	errInvalidNumOfNodes     = errors.New("invalid number of nodes in shard/metachain or in the consensus group")
 	errCreatingKeygen        = errors.New("cannot create key gen")
 )
 
+type txgenAccount struct {
+	PubKey        string   `json:"pubKey"`
+	PrivKey       string   `json:"privKey"`
+	LastNonce     uint64   `json:"lastNonce"`
+	Balance       *big.Int `json:"balance"`
+	TokenBalance  *big.Int `json:"tokenBalance"`
```
Comment (AdoAdoAdo) [inline cmd/filegen/main.go]: what is the difference between balance and token balance?

File: cmd/filegen/main.go (hunk 1 above)
Comment (bogdan-rosianu) [reply cmd/filegen/main.go]: Because they won't be included in PEM files. Only in genesis.json and accounts.json

File: cmd/filegen/main.go (hunk 2 above)
Comment (bogdan-rosianu) [reply cmd/filegen/main.go]: token balance represents the balance inside an ERC20 smart contract

**Pull request merged**